recipes/genozip/meta.yaml

```
@@ -0,0 +1,70 @@
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: genozip
+  version: 1.0.1
+
+source:
+  url: https://github.com/divonlan/genozip/raw/master/conda/genozip-1.0.1.tar.gz
+  sha256: 4828aa3ecb400d0bcf5e554e6fb7bc2be01b104f8b7fc81c63924357ef24aacf
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}        # [not win]
+    - {{ compiler('m2w64_c') }}  # [win]
+    - {{posix}}make
+    - {{posix}}sed               # [win]
+    - {{posix}}coreutils         # [win]
+  #  - make  # [unix]
+    - zlib
+    - bzip2
+  host:
+    - zlib
+    - bzip2
+  run:
+    - {{native}}gcc-libs         # [win]
+    - zlib
+    - bzip2
+
+test:
+  source_files: 
+    - test-file.vcf
+
+  commands:
+    {% set genozip_executables = [
+      "genozip",
+      "genounzip",
+      "genocat",
+      "genols"
+    ] %}
+    {% for executable in genozip_executables %}
+    - test -f ${PREFIX}/bin/{{ executable }}  # [unix]
+    {% endfor %}
+    - genozip --password abc --test test-file.vcf  # [unix]
+
+about:
+  home: https://github.com/divonlan/genozip
+  dev_url: https://github.com/divonlan/genozip
+  license: Apache-2.0, non-commercial only
```

The modification you've made means it's no longer an Apache licence. People should be able to filter on this metadata to whitelist which licences they find acceptable.
```suggestion
  license: CUSTOM
```

---

agreed